app/src/test/java/com/lastaoutdoor/lasta/data/api/ApiDataClassesTest.kt

```diff
@@ -0,0 +1,215 @@
+package com.lastaoutdoor.lasta.data.api
+
+import androidx.test.core.app.ApplicationProvider
+import com.google.api.Context
+import com.lastaoutdoor.lasta.data.model.activity.ActivityType
+import com.lastaoutdoor.lasta.data.model.api.Bounds
+import com.lastaoutdoor.lasta.data.model.api.Node
+import com.lastaoutdoor.lasta.data.model.api.Relation
+import com.lastaoutdoor.lasta.data.model.api.SimpleWay
+import com.lastaoutdoor.lasta.data.model.api.Tags
+import com.lastaoutdoor.lasta.data.model.api.Way
+import com.lastaoutdoor.lasta.data.model.map.Position
+import com.lastaoutdoor.lasta.di.AppModule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class ApiDataClassesTest {
+  private val con = ApplicationProvider.getApplicationContext<android.content.Context>()
+  private val rep = OutdoorActivityRepositoryImpl(AppModule.provideAPIService(con))
```

Comment:
Maybe it would be more optimal to set those values as `private lateinit` and then assign their values in a `@Before` set up function, but that's not urgent